examples/BUILD

```diff
@@ -93,6 +93,41 @@ py_binary(
     ],
 )
 
+py_binary(
```

Comment:
this test should be optional and only work if RSS is included

Reply:
Do you mean something like [this](https://stackoverflow.com/questions/49099181/dont-include-a-cc-library-to-be-built-by-default-unless-explicitly-specified)? In our case, like if a user doesn't run the py_binary or build bark with `--config=rss` ([ref](https://docs.bazel.build/versions/master/guide.html#--config)), then rss will not be build

Reply:
It is just debugging code and it is removed now